implementations/rust/ockam/ockam_command/src/util/output.rs

```diff
@@ -182,16 +182,14 @@ impl Output for ShowSecureChannelResponse<'_> {
         let s = match &self.channel {
             Some(addr) => {
                 format!(
-                    "\n  Secure Channel:\n{} {}\n{} {}\n{} {}\n{} {}",
+                    "\n  Secure Channel:\n{} {}\n{} {}\n{} {}",
                     "  •         At: ".light_magenta(),
                     route_to_multiaddr(&route![addr.to_string()])
                         .context("Invalid Secure Channel Address")?
                         .to_string()
                         .light_yellow(),
                     "  •         To: ".light_magenta(),
                     self.route.as_ref().unwrap().light_yellow(),
-                    "  • Identifier: ".light_magenta(),
```

Comment:
@mrinalwadhwa it was initiator id, not responder (so, always the same and equals to nodes id). I removed it during resolving conflicts, tell me if you want it back